src/App.js

```diff
@@ -14,11 +16,56 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
```

Comment:
ты не используешь это как массив, зачем он тут?
```suggestion
    selectedGoods: null,
```